content/shared/v3-core-get-started/_index.md

```diff
@@ -252,15 +252,40 @@ To have the `influxdb3` CLI use your admin token automatically, assign it to the
 
 To create an admin token, use the `influxdb3 create token --admin` subcommand--for example:
 
+{{% code-placeholders "INFLUXDB_HOST|CONTAINER_NAME" %}}
+
+{{< code-tabs-wrapper >}}
+
+{{% code-tabs %}}
+[CLI](#)
+[Docker](#)
+{{% /code-tabs %}}
+
+{{% code-tab-content %}}
 ```bash
 influxdb3 create token --admin \
- --host http://{{< influxdb/host >}}
+  --host http://INFLUXDB_HOST
```

Comment:
Keep the `influxdb/host` shortcode here. It should auto-populate based on the the product and allow users to update it.